react-main/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,15 +1,38 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+
+import { calculateWinner } from '../../../../utils';
```

Comment:
Since this function is only used by this component, i would place it inside this component's folder